app/src/main/res/layout/fragment_view_video.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Oh noes, pls don't make a new `RelativeLayout`, they're deprecated and slow. Please use `ConstraintLayout`